src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -18,8 +18,16 @@ import ProteinImpactTypePanel from "../../../shared/components/mutationTypePanel
 import ProteinChainPanel from "../../../shared/components/proteinChainPanel/ProteinChainPanel";
 import {computed, action, observable} from "mobx";
 
+export interface IMutationMapperConfig {
```

Comment:
This type is introduced in order to avoid directly including `AppConfig` within `MutationMapper`. Eventually, we will separate `MutationMapper` into a standalone library. In that case we cannot use the global `AppConfig`.